docs/troubleshooting.md

```diff
@@ -10,6 +10,16 @@ installed and configured:
 - Ensure not running in a containerized environment (for non-production environments see [this post](https://developers.redhat.com/blog/2014/05/05/running-systemd-within-docker-container/) for a workaround)
 - Check installation logs for more details
 
+## HTTP Error Codes
+
+- 401 (UNAUTHORIZED): Configured access token or realm is incorrect
+- 404 (NOT FOUND): Likely configuration parameter is wrong like endpoint or URI
+  (e.g. /v1/log); possible network/firewall/port issue
+- 429 (TOO MANY REQUESTS): Org is not provisioned for the amount of traffic
+  being sent; reduce traffic or request increase in capacity
+- 503 (SERVICE UNAVAILABLE): If using the Log Observer, this is the same as 429.
+  (because that is how HECv1 responds)
```

Comment:
```suggestion
- 503 (SERVICE UNAVAILABLE): If using the Log Observer, this is the same as 429
  (because that is how HECv1 responds). Otherwise, check the status page.
```